app.js

```diff
@@ -84,6 +83,8 @@ oppiaGithubPages.config(['$routeProvider', function($routeProvider) {
     .otherwise({
       redirectTo: '/'
     });
+     //Use the HTML5 History API to prettfy URL(by removing #)
```

Comment:
typo: prettify
nit: add space before '('

(also, nice addition! That # was annoying)